versions/1.1/SPEC.md

```diff
@@ -0,0 +1,5775 @@
+
+1. They are of the same type.
+2. They are the same length.
+3. All of their contained elements are equal.
+
+Since `Array`s and `Map`s are ordered, the order of their elements are also compared. For example:
+
+```wdl
+[1, 2, 3] == [1, 2, 3]
+[1, 2, 3] != [2, 1, 3]
+{"a": 1, "b": 2} != {"b": 2, "a": 1}
+```
+
+Note that [type coercion](#type-coercion) can be employed to compare values of different but compatible types. For example:
+
+```wdl
+Array[Int] i = [1,2,3]
+Array[Float] f = [1.0, 2.0, 3.0]
+i == f  # false
+
+# coerce i to an array of Float
+Array[Float] i_to_f = i
+i_to_f == f  # true
+```
+
+##### Equality and Inequality Comparison of Optional Types
+
+The equality and inequality operators are exceptions to the general rules on [coercion of optional types](#coercion-of-optional-types). Either or both operands of an equality or inequality comparison can be optional, considering that `None` is equal to itself but no other value.
+
+```wdl
+Int i = 1
+Int? j = 1
+Int? k = None
+
+i == j  # true
+i == k  # false
+j != k  # true
+k == None  # true
+```
+
+#### Operator Precedence Table
+
+| Precedence | Operator type         | Associativity | Example              |
+|------------|-----------------------|---------------|----------------------|
+| 11         | Grouping              | n/a           | (x)                  |
+| 10         | Member Access         | left-to-right | x.y                  |
+| 9          | Index                 | left-to-right | x[y]                 |
+| 8          | Function Call         | left-to-right | x(y,z,...)           |
+| 7          | Logical NOT           | right-to-left | !x                   |
+|            | Unary Negation        | right-to-left | -x                   |
+| 6          | Multiplication        | left-to-right | x*y                  |
+|            | Division              | left-to-right | x/y                  |
+|            | Remainder             | left-to-right | x%y                  |
+| 5          | Addition              | left-to-right | x+y                  |
+|            | Subtraction           | left-to-right | x-y                  |
+| 4          | Less Than             | left-to-right | x<y                  |
+|            | Less Than Or Equal    | left-to-right | x<=y                 |
+|            | Greater Than          | left-to-right | x>y                  |
+|            | Greater Than Or Equal | left-to-right | x>=y                 |
+| 3          | Equality              | left-to-right | x==y                 |
+|            | Inequality            | left-to-right | x!=y                 |
+| 2          | Logical AND           | left-to-right | x&&y                 |
+| 1          | Logical OR            | left-to-right | x\|\|y               |
+
+#### Member Access
+
+The syntax `x.y` refers to member access. `x` must be a call in a workflow, or a struct or object value. A call can be thought of as a struct where the members are the outputs of the called task.
```

Comment:
Slight formatting/caveat change:
"The syntax `x.y` refers to member access. `x` must be either a call in a workflow, a `struct` or an `object` (until it is deprecated). A call can be thought of as a `struct` where the members are the outputs of the called task."